src/App.tsx

```diff
@@ -1,24 +1,124 @@
+import { Filter } from './components/Filter/Filter';
+import { Item } from './components/Todo/Todo';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 6470;
 
 export const App: React.FC = () => {
+  const [titleTodo, setTitleTodo] = useState('');
+  const [listTodo, setListTodo] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [tempTodo, setTempTodo] = useState(false);
+
+  const getListTodo = async (filter?: boolean) => {
+    try {
+      const result = await getTodos(USER_ID, filter);
+
+      setListTodo(result);
+    } catch (e) {
+      setError('Oops, something were wrong, please try again later');
+    }
+  };
+
+  useEffect(() => {
+    getListTodo();
+  }, []);
+
+  const clearError = useCallback(() => setError(''), []);
+
+  const onSubmitHandler = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setTempTodo(true);
+    try {
+      await postTodo(
+        { title: titleTodo, userId: USER_ID, completed: false },
+      );
+
+      getListTodo();
+      setTitleTodo('');
+      setTempTodo(false);
+    } catch (e) {
+      setError('Oops, something were wrong, please try again later');
+    }
+  };
+
+  const onChangeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitleTodo(event.target.value);
+  };
+
+  const deleteTodoHandler = (id: number) => {
+    deleteTodo(id).then(() => getListTodo());
+  };
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          {/* this buttons is active only if there are some active todos */}
+          <button type="button" className="todoapp__toggle-all active" />
+
+          {/* Add a todo on form submit */}
+          <form onSubmit={onSubmitHandler}>
+            <input
+              type="text"
+              className="todoapp__new-todo"
+              placeholder="What needs to be done?"
+              value={titleTodo}
+              onChange={onChangeHandler}
+            />
+          </form>
+        </header>
+
+        <section className="todoapp__main">
+          {
+            listTodo.map((el: Todo) => (
+              <Item
+                todo={el}
+                key={el.id}
+                deleteTodo={deleteTodoHandler}
+              />
+            ))
+          }
+          {tempTodo && (
+            <Item
+              todo={{
+                id: Math.random(),
```

Comment:
Not a good way to create an id. You can use different libraries for it
https://dev.to/salehmubashar/create-a-unique-id-in-react-js-3f75